### ISSUE TITLE
Social Media links should be editable via business details. 

### ISSUE THREAD

**Issue by ermatitan** [titan-cebu/Core-Cider#5]
![image](https://cloud.githubusercontent.com/assets/12791909/10092630/3810385a-637b-11e5-9435-bb488a3ae04e.png)

**Comment by bernardtitan** [comment]
If this is not possible, kindly say so

**Comment by ricky-titan** [comment]
yep it is not a custom theme so all the settings have to go through theme
